README.md

```diff
@@ -18,26 +28,24 @@ reducing disk seeks and network calls in a cluster.
 
 
 Dgraph's goal is to provide [Google](https://www.google.com) production level scale and throughput,
-with low enough latency to be serving real-time user queries, over terabytes of structured data.
-Dgraph supports [GraphQL query syntax](https://dgraph.io/docs/master/query-language/), and responds in [JSON](http://www.json.org/) and [Protocol Buffers](https://developers.google.com/protocol-buffers/) over [GRPC](http://www.grpc.io/) and HTTP.
+with low enough latency to serve real-time user queries over terabytes of structured data.
+Dgraph supports [GraphQL query syntax](https://dgraph.io/docs/master/query-language/), and responds in [JSON](http://www.json.org/) and [Protocol Buffers](https://developers.google.com/protocol-buffers/) over [GRPC](http://www.grpc.io/) and HTTP. Dgraph is written using the Go Programming Language.
 
 **Use [Discuss Issues](https://discuss.dgraph.io/c/issues/dgraph/38) for reporting issues about this repository.**
```

Comment:
We can remove this right, since we are using Github issues?  (It has already been removed from the readme's in Badger, Ristretto).